hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/TokenStateTranslator.java

```diff
@@ -214,4 +220,12 @@ static List<com.hedera.node.app.service.mono.state.submerkle.FcCustomFee> conver
 
         return monoCustomFees;
     }
+
+    private static AccountID asAccount(final long num) {
+        return AccountID.newBuilder().accountNum(num).build();
+    }
+
+    private static TokenID asToken(final long num) {
+        return TokenID.newBuilder().tokenNum(num).build();
+    }
```

Comment:
These methods already exist in `BaseCryptoHandler`

Reply:
mono-service shouldn't use BaseCryptoHandler as it would create circular dependencies.

Reply:
Causes a circular dependency, so let's leave these for now.